client/src/components/Learning/Content/Content.js

```diff
@@ -14,11 +14,26 @@ export default function Content(props) {
     });
 
     if (headings.length < 1) setChapterIndex([]);
+
+    const codingQuestions = document.querySelectorAll("blockquote a");
```

Comment:
Adding few links for you to refer, to know why and when and if you should use query selector in React.  
https://stackoverflow.com/questions/59198952/using-document-queryselector-in-react-should-i-use-refs-instead-how
https://www.webtips.dev/solutions/using-queryselector-in-react

Reply:
done